src/Microsoft.Framework.PackageManager/Bundle/BundleProject.cs

```diff
@@ -507,5 +501,63 @@ private string BasePath(string relativePath)
             var index2 = (relativePath + Path.AltDirectorySeparatorChar).IndexOf(Path.AltDirectorySeparatorChar);
             return relativePath.Substring(0, Math.Min(index1, index2));
         }
+
+        private Runtime.Project ResolveProject(string name)
+        {
+            Runtime.Project project;
+            if (!_projectResolver.TryResolveProject(name, out project))
+            {
+                throw new Exception("TODO: unable to resolve project named " + name);
```

Comment:
I know it is unrelated but I think we should start addressing these TODO error messages